src/components/map/layers/VectorGridLayer.tsx

```diff
@@ -0,0 +1,31 @@
+import L from 'leaflet';
+import 'leaflet.vectorgrid';
+import { GridLayer, GridLayerProps, withLeaflet } from 'react-leaflet';
+
+declare module 'leaflet' {
+    let vectorGrid: any;
+}
+
+interface IVectorGridLayerProps extends GridLayerProps {
+    url: string;
+    tms: boolean;
+    vectorTileLayerStyles: any;
```

Comment:
Is this variable used?

Reply:
Check: https://github.com/HSLdevcom/jore-map-ui/pull/234/files#diff-612cdc0c79c52d292f22d01e01e4ed78R101